tendrl/node_agent/manager/pull_service_status.py

```diff
@@ -1,14 +1,14 @@
 from tendrl.commons.utils.service_status import ServiceStatus
 
 TENDRL_SERVICE_TAGS = {
-    "tendrl-node-agent": "TENDRL",
-    "etcd": "TENDRL_SERVER",
-    "tendrl-apid": "TENDRL_SERVER",
-    "tendrl-gluster-integration": "TENDRL_GLUSTER",
-    "tendrl-ceph-integration": "TENDRL_CEPH",
-    "glusterd": "GLUSTER",
-    "ceph-mon": "CEPH_MON",
-    "ceph-osd": "CEPH_OSD"
+    "tendrl-node-agent": "TENDRL/NODE",
```

Comment:
Should we really keep the names uppercase? I feel these would becomes keys in etcd as well. So we might make them lowercase for better typing.. suggestions?